### ISSUE TITLE
About the usage

### ISSUE THREAD

**Issue by will46616** [asteroid-team/asteroid#667]
HI, I have downloaded LibriMix dataset, could I use the model on Asteroid to train on my own dataset?

**Comment by mpariente** [comment]
Yes, just write your own Dataset object and use it in one of the recipes.
